Resources/Locale/en-US/ensnare/ensnare-component.ftl

```diff
@@ -1,5 +1,5 @@
-﻿ensnare-component-try-free = You struggle to remove {$ensnare} that's ensnaring you!
-ensnare-component-try-free-complete = You successfully free yourself from the {$ensnare}!
-ensnare-component-try-free-fail = You fail to free yourself from the {$ensnare}!
+﻿ensnare-component-try-free = Você tenta remover {$ensnare} que está te prendendo!
+ensnare-component-try-free-complete = Você tem sucesso em se libertar de {$ensnare}!
+ensnare-component-try-free-fail = VocÊ falha em se libertar de {$ensnare}!
 
-ensnare-component-try-free-other = You start removing the {$ensnare} caught on {$user}!
+ensnare-component-try-free-other = Você começa remover {$ensnare} preso n{ THE($user) }!
```

Comment:
Aqui está correto? "preso n{ THE($user) }!"

Reply:
teoricamente sim